### PR TITLE
Jetpack Gutenberg Blocks Preset: :abc: OCD

### PR DESCRIPTION
**Pull request by ockham**
#### Changes proposed in this Pull Request

* Alphabetize the beta section of `index.json`. Rebases have been driving me nuts. If we stabilize on alphabetic order, they should become less of a pain,

#### Testing instructions

Everything still there? In alphabetic order? No comma on the last line? (JSON!)

### CODE REVIEW

File: client/gutenberg/extensions/presets/jetpack/index.json [hunk 1]
```diff
@@ -12,12 +12,12 @@
...
-    "mailchimp",
+    "business-hours",
     "contact-info",
-    "wordads",
+    "mailchimp",
+    "slideshow",
     "videopress",
     "vr",
-    "business-hours",
-    "slideshow"
```
Comment (sirreal) [inline client/gutenberg/extensions/presets/jetpack/index.json]: This was like a pig sty!

File: client/gutenberg/extensions/presets/jetpack/index.json [hunk 2]
```diff
@@ -12,12 +12,12 @@
     "tiled-gallery"
   ],
   "beta": [
-    "mailchimp",
+    "business-hours",
```
Comment (sirreal) [inline client/gutenberg/extensions/presets/jetpack/index.json]: 🙂 Sanity!

**Comment by lezama** [comment]
now we can sleep in peace, thanks Bernie!

File: client/gutenberg/extensions/presets/jetpack/index.json (hunk 1 above)
Comment (ockham) [reply client/gutenberg/extensions/presets/jetpack/index.json]: :pig:

**Pull request merged**